bench/zarith/benchZarith.ml

```diff
@@ -0,0 +1,47 @@
+(* dune exec bench/zarith/benchZarith.exe -- -a *)
```

Comment:
Maybe we can add a comment here on what exactly this is benchmarking and why it is needed?